autonomy/fsm/scaffold/generators/components.py

```diff
@@ -152,15 +152,13 @@ class PayloadsFileGenerator(AbstractFileGenerator, PAYLOADS):
     def _get_base_payload_section(self) -> str:
         """Get the base payload section."""
 
-        payloads: List[str] = [self.BASE_PAYLOAD_CLS.format(**self.template_kwargs)]
+        payloads: List[str] = []
```

Comment:
could be removed but task here is not to fix or optimize the scaffolding stuff, that is of secondary concern as long as we ensure it remains compatible with the new design (which is now is)